doc/mgr/telemetry.rst

```diff
@@ -64,53 +64,151 @@ the way Ceph is used.
 
 Data is sent secured to *https://telemetry.ceph.com*.
 
+Individual channels can be enabled or disabled with::
+
+  ceph telemetry enable channel basic
+  ceph telemetry enable channel crash
+  ceph telemetry enable channel device
+  ceph telemetry enable channel ident
+  ceph telemetry enable channel perf
+
+  ceph telemetry disable channel basic
+  ceph telemetry disable channel crash
+  ceph telemetry disable channel device
+  ceph telemetry disable channel ident
+  ceph telemetry disable channel perf
+
+Multiple channels can be enabled or disabled with::
+
+  ceph telemetry enable channel basic crash device ident perf
+  ceph telemetry disable channel basic crash device ident perf
+
+Please note that telemetry should be on for these commands to take effect.
+
+List all channels with::
+
+  ceph telemetry channel ls
+
+  NAME      ENABLED    DEFAULT    DESC
+  basic     ON         ON         Share basic cluster information (size, version)
+  crash     ON         ON         Share metadata about Ceph daemon crashes (version, stack straces, etc)
+  device    ON         ON         Share device health metrics (e.g., SMART data, minus potentially identifying info like serial numbers)
+  ident     OFF        OFF        Share a user-provided description and/or contact email for the cluster
+  perf      ON         OFF        Share perf counter metrics summed across the whole cluster
+
+
+Enabling Telemetry
+------------------
+
+To allow the *telemetry* module to start sharing data::
+
+  ceph telemetry on
```

Comment:
are the 'non-default' channels not enabled by this? If so that should be mentioned here - so people know they exist if they've just gone straight to this section and haven't read the doc from the top

Reply:
> are the 'non-default' channels not enabled by this? If so that should be mentioned here - so people know they exist if they've just gone straight to this section and haven't read the doc from the top

@jdurgin Non-default channels are not enabled with this. When the user turns telemetry on in the CLI, a message is printed that says something like "Channel <channel name> is turned off by default. Consider enabling it with <command>." So it's made clear to the user in that regard, but perhaps it would still be good to make mention of this in the doc.